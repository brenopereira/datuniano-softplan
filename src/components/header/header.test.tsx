import React from 'react';
import renderer from 'react-test-renderer';

import { Header } from '.';

test('Validate render Header component', () => {
    const component = renderer.create(<Header title='Softplan' />);

    console.log(component.toJSON());
});
