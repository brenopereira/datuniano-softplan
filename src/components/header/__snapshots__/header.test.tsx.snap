// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validate render Header component 1`] = `
<div>
  Softplan
</div>
`;
